packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -699,13 +710,16 @@ class EditorPage {
 		const elementName = isAndroid() ? '//*' : '//XCUIElementTypeButton';
 
 		const locator = `${ elementName }[starts-with(@${ this.accessibilityIdXPathAttrib }, "Button position")]`;
-		await this.driver.elementByXPath( locator ).click();
+		let optionMenuButton = await waitForVisible( this.driver, locator );
+		await optionMenuButton.click();
 
 		const optionMenuButtonLocator = `${ elementName }[contains(@${ this.accessibilityIdXPathAttrib }, "${ buttonPosition }")]`;
-		return await this.driver
-			.elementByXPath( optionMenuButtonLocator )
-			.click()
-			.sleep( isAndroid() ? 600 : 200 ); // sleep a little longer due to multiple menus.
+		optionMenuButton = await waitForVisible(
+			this.driver,
+			optionMenuButtonLocator
+		);
+
+		return await optionMenuButton.click();
```

Comment:
I think we could also use here the new helper `clickIfClickable`, although we'd need to make the `clickIfClickable` function return the element.